integration_tests/pages/overview/OverviewPage.ts

```diff
@@ -1,31 +1,14 @@
 import Page, { PageElement } from '../page'
 import CreateGoalPage from '../goal/CreateGoalPage'
 import UpdateGoalPage from '../goal/UpdateGoalPage'
+// eslint-disable-next-line import/no-cycle
+import FunctionalSkillsPage from '../functionalSkills/FunctionalSkillsPage'
 
 export default class OverviewPage extends Page {
   constructor() {
     super('overview')
   }
 
-  hasBreadcrumb() {
```

Comment:
I've moved the breadcrumb functions to the `Page` superclass, because many (most) pages display the breadcrumb bar, and the cypress tests for any page could make use of these methods (as is the case with the `FunctionalSkillsPage`)